### DISCUSSION TITLE
Mouseover needed for status smiley

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#125982] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

See that smiley in the center of the image?
![Screenshot 2024-05-27 19 10 29](https://github.com/community/community/assets/1230959/4a386ab2-9f9e-4e44-becd-20953a5d915c)
Well when you put your mouse on it nothing pops up explaining what it is for. The only way to find out is to... click it!

**Comment by hexadecimal233** [comment]
I tried opening my profile when I am logged out, and everything works fine.
It breaks when I have account logged in, and if it is not yours, this glitch won't appear.
